src/Components/Blazor/BlazorExtension/src/Microsoft.VisualStudio.BlazorExtension.csproj

```
@@ -1,23 +1,35 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
+    <TargetFramework>net472</TargetFramework>
+
+    <!-- Use the same experimental hive as Roslyn and Razor. This makes it easy to mix private builds. -->
+    <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
+
+    <!-- Required to run the project localy -->
+    <StartAction>Program</StartAction>
+    <StartProgram>$(DevEnvDir)devenv.exe</StartProgram>
+    <StartArguments>/rootsuffix $(VSSDKTargetPlatformRegRootSuffix) /log</StartArguments>
+
+    <!--
+      Mark the VSIX as a per-computer install (not-per-user). Putting a component "in the box"
+      requires this, and trying to change it after doing a release has lot of problems.
+    -->
+    <ExtensionInstallationRoot>CommonExtensions</ExtensionInstallationRoot>
+    <ExtensionInstallationFolder>Microsoft\Blazor</ExtensionInstallationFolder>
+
+    <!-- This should be set as true if we're ever building the VSIX for inclusion by the VS installer. -->
+    <IsProductComponent Condition="'$(IsProductComponent)'==''">false</IsProductComponent>
```

```suggestion
    <IsProductComponent>false</IsProductComponent>
```

Removing the condition so this isn't accidentally overridden some how.